simpleratingbar/src/main/java/com/iarcuschin/simpleratingbar/SimpleRatingBar.java

```diff
@@ -673,6 +677,56 @@ private void setNewRatingFromTouch(float x, float y) {
       }
     }
   }
+  
+  @Override
+  protected Parcelable onSaveInstanceState() {
+      Parcelable superState = super.onSaveInstanceState();
+      SavedState savedState = new SavedState(superState);
+      savedState.rating = getRating();
+      return savedState;
+    }
+
+  @Override
+  protected void onRestoreInstanceState(Parcelable state) {
+      SavedState savedState = (SavedState) state;
+      super.onRestoreInstanceState(savedState.getSuperState());
+      setRating(savedState.rating);
```

Comment:
Wouldn't this trigger a call to onRatingChanged method of listener every time the view state is restored ?
I think it would be better to set the rating and invalidate the view without calling `setRating` method.